### PR TITLE
Queue with stacks

### PR DESCRIPTION
**Pull request by ClementBuchanan**
- [x] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: Challenges/queueWithStacks/queue-with-stacks.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const Stack = require('.stacks.js');
+
+class Queue {
```
Comment (lizkavalski) [inline Challenges/queueWithStacks/queue-with-stacks.js:5]: rename to queueWithStacks
